src/app/rundown-view/components/rundown-subheader/rundown-subheader.component.html

```diff
@@ -0,0 +1,23 @@
+<sofie-spacer></sofie-spacer>
+
+<div class="subheader-panel">
+    <span class="subheader-panel__label">Part countdown</span>
+    <span class="subheader-panel__value">00:04</span>
+</div>
+
+<div class="subheader-panel">
+    <span class="subheader-panel__label">Segment countdown</span>
+    <span class="subheader-panel__value">00:12</span>
+</div>
+
+<div class="subheader-panel">
+    <span class="subheader-panel__label">Segment timer</span>
+    <span class="subheader-panel__value">00:08</span>
+</div>
+
+<div class="subheader-panel end-words">
+    <span class="subheader-panel__label">End words</span>
+    <span class="subheader-panel__value">{{ endWords }}</span>
+</div>
+
+<sofie-spacer></sofie-spacer>
```

Comment:
There is not really anything in here that is header specific, so there is no reason to call this `subheader`. 
This indicates that you have named the component from where you going to use it, instead of what it does.

What it does is display varius timings and the end words. 
If you can't find a fitting name for that then maybe that shows that this shouldn't a component on its own? Or at the very least, this is the wrong place to separate the component?

A further indications that this seems to be developed with where it is going to be used instead of what it does is the usages of `SofieSpacer`.
Why do the component need to pre- and postpend space to itself? My guess is to make it look nice where it is used. But the component is not allowed to know anything about where it is used...

Reply:
It is not header specific, but subheader specific ;) And on a serious note: I agree that subheader is a bad. I have changed it to `OnAirDetailsPanel`.

A note regarding the spacer. It is not padding the component itself. The spacer is placed inside the component and is used to restrict the space available to the sub panels.